cdap-ui/cypress/integration/pipeline.canvas.actions.spec.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import {
+  loginIfRequired,
+  getGenericEndpoint,
+  getConditionNodeEndpoint,
+  getArtifactsPoll
+} from '../helpers';
+import { DEFAULT_GCP_PROJECTID, DEFAULT_GCP_SERVICEACCOUNT_PATH, DEFAULT_BIGQUERY_DATASET, DEFAULT_BIGQUERY_TABLE } from '../support/constants';
+import { INodeInfo, INodeIdentifier } from '../typings';
+let headers = {};
+
+describe('Pipeline Canvas actions', () => {
+  // Uses API call to login instead of logging in manually through UI
+  before(() => {
+    loginIfRequired().then(() => {
+      cy.getCookie('CDAP_Auth_Token').then((cookie) => {
+        if (!cookie) {
+          return;
+        }
+        headers = {
+          Authorization: 'Bearer ' + cookie.value,
+        };
+      });
+    });
+  });
+
+  beforeEach(() => {
+    getArtifactsPoll(headers);
+  });
+
+  it.skip('Should correctly undo/redo actions done by the user', () => {
```

Comment:
why is this one skipped?

Reply:
🤦‍♂ development changes. Will revert it back.